e2e/app/modal/workspace-review-research-purpose-modal.ts

```diff
@@ -0,0 +1,26 @@
+import Modal from './modal';
+import { Page } from 'puppeteer';
+import { waitForText } from 'utils/waits-utils';
+import { LinkText } from 'app/text-labels';
+
+const modalTitle = 'Please review Research Purpose for Workspace';
+
+export default class WorkspaceReviewResearchPurposeModal extends Modal {
```

Comment:
Added function and test to cover `Review Research Purpose` workflow.

![Screen Shot 2021-04-16 at 6 14 18 PM](https://user-images.githubusercontent.com/35533885/115089152-ad75b480-9edf-11eb-847f-f2646e707a08.png)